packages/hoprd/src/api/v3/paths/tickets/price.ts

```diff
@@ -0,0 +1,66 @@
+import type { Operation } from 'express-openapi'
+import { STATUS_CODES } from '../../utils.js'
+import { Hopr } from '@hoprnet/hopr-utils'
+
+
+const GET: Operation = [
+  async (req, res, _next) => {
+    const { node }: { node: Hopr } = req.context
+
+    try {
+      const ticket_price = await node.getTicketPrice()
+      return res.status(200).send({price: ticket_price})
+    } catch (err) {
+      return res
+        .status(422)
+        .send({ status: STATUS_CODES.UNKNOWN_FAILURE, error: err instanceof Error ? err.message : 'Unknown error' })
+    }
+  }
+]
+
+GET.apiDoc = {
+  description: 'Get the latest ticket price.',
+  tags: ['Tickets'],
+  operationId: 'ticketsGetTicketPrice',
+  responses: {
+    '200': {
+      description: 'Price fetched successfully.',
+      content: {
+        'application/json': {
+          schema: {
+            type: 'object',
+            properties: {
+              price: {
```

Comment:
The access function to the ticket price in the db has no defaults. Should the database access function return the default `DEFAULT_PRICE_PER_PACKET` set in `core-protocol` ?

Reply:
U256 is in Wei. It'd be nice to have a proper unit system around these values... so that they can be visibly handled properly, e.g. unit system with `Base / 1e18` = `Wei`... etc.

Reply:
> The access function to the ticket price in the db has no defaults. Should the database access function return the default `DEFAULT_PRICE_PER_PACKET` set in `core-protocol` ?

No, the DB without the value is a bug. It should not happen, that's when the API endpoint in TS can fail with 422, if is sees undefined. Do not inject the defaults. They are not necessary.